Modules/Core/Common/include/itkConceptChecking.h

```diff
@@ -852,7 +852,7 @@ template <typename T>
 struct IsInteger
 {
   using Self = IsInteger;
-  static constexpr bool Integral = NumericTraits<T>::IsInteger;
+  static constexpr bool Integral = std::is_integral<T>::value;
```

Comment:
C++14 offers a few alternatives to the form `std::is_integral<T>::value`, like:

    static constexpr bool Integral2 = std::is_integral<T>();
    static constexpr bool Integral3 = std::is_integral<T>{};
    static constexpr bool Integral4 = std::is_integral<T>{}();

But I think `std::is_integral<T>::value` is just fine, here 👍